test/tbb/test_task_arena.cpp

```diff
@@ -1748,7 +1748,13 @@ struct enqueue_test_helper {
 
 //--------------------------------------------------//
 
-// This test should be first
+//! Test for uninitilized arena
+//! \brief \ref requirement \ref error_guessing
+TEST_CASE("Test current_thread_index") {
```

Comment:
As I understand this test should be performed first to work correctly. But I cannot find information about test cases order inside the DocTest documentation. What do you think about adding some protection mechanism which will define the cases in the right order or just showing the message and fails if the test cases order is incorrect?

As I understand, the order of test cases may be set using [subcases](https://github.com/onqtam/doctest/blob/master/doc/markdown/tutorial.md#test-cases-and-subcases).

Reply:
I added error message: "TBB was initialized".